CHANGELOG.md

```diff
@@ -83,6 +85,10 @@
 
     In v1, these methods should be defined only on the `Component.View` class instead.
 
+#### Refactor
+
+- `Component.get_context_data()` can now omit a return statement or return `None`.
```

Comment:
This is to allow to omit the `return` statement, e.g.

```py
class Table(Component):
    def get_context_data(self):
        assert self.input.args == [1, 2]
```

I think it makes sense to allow `get_context_data()` to not return an object (AKA return `None`), because I've found myself many times just trying some logic within that method, but without really any need to return any data.